lang/csharp/src/apache/main/CodeGen/CodeGen.cs

```diff
@@ -92,6 +80,17 @@ public CodeGen()
             this.Schemas = new List<Schema>();
             this.Protocols = new List<Protocol>();
             this.NamespaceMapping = new Dictionary<string, string>();
+            this.NamespaceLookup = new Dictionary<string, CodeNamespace>();
```

Comment:
Do we need to pass `StringComparer.Ordinal` as in line 71 above ?

Reply:
Yes good catch.